recipe/patches/0001-skip-failures.patch

```diff
@@ -0,0 +1,368 @@
+From 18f295f9ad9edcf3858ed6efab07a9fa02f8c0e0 Mon Sep 17 00:00:00 2001
+From: Mohamed Sentissi <msentissi@anaconda.com>
+Date: Thu, 24 Oct 2024 13:40:23 -0400
+Subject: [PATCH] skip failures
+
```

Comment:
```suggestion

We patch out some tests because they expect pandoc 3.2 as a dependency (the version on CF), but we have an earlier version on defaults.

One failed because it needs a PDF engine such as pdflatex, which is optional.

````